lib/api/errors/parse_error.rb

```diff
@@ -30,6 +30,8 @@
 module API
   module Errors
     class ParseError < InvalidRequestBody
+      identifier InvalidRequestBody.identifier
```

Comment:
This one is quite tricky. Inheritance does not work for identifiers.

I chose the lazy approach of simply declaring to use the same identifier as the parent class. The alternative would be to change `identifier` so that it implicitly "inherits" from the superclass, however I think the explicit approach might be more sensible here.